modules/nw-pod-network-connectivity-configuration.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * networking/verifying-connectivity-endpoint.adoc
+
+[id="nw-pod-network-connectivity-configuration_{context}"]
+= Configuring pod connectivity check placement
+
+As a cluster administrator, you can configure on which nodes the connectivity check pods run by modifying the `network.config.openshift.io` object named `cluster`.
+
+.Prerequisites
+
+* Install the {oc-first}.
+
+.Procedure
+
+. To edit the connectivity check configuration, enter the following command:
++
+[source,terminal]
+----
+$ oc edit network.config.openshift.io cluster
+----
+
+. In the text editor, update the `networkDiagnostics` stanza to specify the node selectors that you want for the source and target pods.
+
+. To commit your changes, save your changes and exit the text editor.
+
+.Verification
+
+To verify that the source and target pods are running on the intended nodes, enter the following command:
+
+[source,terminal]
+----
+$ oc get pods  -n openshift-network-diagnostics  -o wide
+----
+
+.Example output
+[source,text]
+----
+NAME                                    READY   STATUS    RESTARTS   AGE   IP            NODE                                        NOMINATED NODE   READINESS GATES
+network-check-source-75b88967df-mlfsn   0/1     Running   0          43s   <none>        <none>  
```

Comment:
This output looks a little weird, there are no network-check-target pods.
Additionally the source pod is not ready and there is no node assigned.
Example output from a 4.16 cluster with default settings:
```
oc get po -n openshift-network-diagnostics -o wide

NAME                                    READY   STATUS    RESTARTS   AGE   IP            NODE                                                             NOMINATED NODE   READINESS GATES
network-check-source-7cdbc9858d-77bsv   1/1     Running   0          19m   10.128.2.12   pdiak-05-27-2024-xt8qh-worker-b-5f99q                            <none>           <none>
network-check-target-225wg              1/1     Running   0          19m   10.130.0.3    pdiak-05-27-2024-xt8qh-master-2.c.openshift-gce-devel.internal   <none>           <none>
network-check-target-45vgz              1/1     Running   0          19m   10.129.0.3    pdiak-05-27-2024-xt8qh-master-1.c.openshift-gce-devel.internal   <none>           <none>
network-check-target-5vlpx              1/1     Running   0          19m   10.128.0.4    pdiak-05-27-2024-xt8qh-master-0.c.openshift-gce-devel.internal   <none>           <none>
network-check-target-7qj6h              1/1     Running   0          10m   10.128.2.4    pdiak-05-27-2024-xt8qh-worker-b-5f99q                            <none>           <none>
network-check-target-f9r44              1/1     Running   0          10m   10.129.2.4    pdiak-05-27-2024-xt8qh-worker-a-w6qdk                            <none>           <none>
network-check-target-r92nl              1/1     Running   0          10m   10.131.0.3    pdiak-05-27-2024-xt8qh-worker-c-hxkd4                            <none>           <none>
```